TestResultSummaryService/plugins/DataManagerPerf.js

```diff
@@ -6,8 +6,10 @@ module.exports.onBuildDone = async (task, { testResultsDB, logger }) => {
     logger.debug("onBuildDone", task.buildName);
     if ( task.type === "Perf" ) {
         if ( !task.aggregateInfo ) {
-            let benchmarkName, benchmarkVariant, jdkDate, javaVersion, nodeVersion, nodeRunDate;
-            const metricsCollection = {};
+            let jdkDate, javaVersion, nodeRunDate, nodeVersion;
+            let aggRawMetricValuesOfChildren = [];
```

Comment:
As I mentioned in `aggDataCollect` in DataManagerAggregate.js that we should make `aggRawMetricValues` a JSON object instead of an array since that change could simplify some the logic here. Similarly, we should do the same for `aggRawMetricValuesOfChildren`.